draft-esdih-ccamp-layer0-types-ext-00.txt

```diff
@@ -0,0 +1,1344 @@
+           description "The current channel received power ";
+       }
+
+       leaf rx-total-power {
+           type dbm-t;
+           config false;
+           description "Current total received power";
+       }
+
+     } // grouping for configured attributes out of mode
+
+     grouping l0-tunnel-attributes {
+       description
+         "Parameters for Layer0 (WSON or Flexi-Grid) Tunnels.";
+       leaf fec-type {
+         type identityref {
+           base fec-type;
+         }
+         description
+           "FEC type.";
+       }
+       leaf termination-type {
+         type identityref {
+           base term-type;
+         }
+         description
+           "Termination type.";
+       }
+       leaf bit-stuffing {
+         type boolean;
+         description
+           "Bit stuffing enabled/disabled.";
+       }
+     }
+
+     grouping l0-path-constraints {
+       description
+         "Global named path constraints configuration
+          grouping for Layer0 (WSON or Flexi-Grid) paths.";
+       leaf wavelength-assignment {
+         type identityref {
+           base wavelength-assignment;
+         }
+         description "Wavelength Allocation Method";
+       }
+     }
+
+
+
+Beller, et al.           Expires 26 August 2021                [Page 21]
+
+Internet-Draft         Layer 0 Types - Revision 2          February 2021
+
+
+   }
+   <CODE ENDS>
+
+                                  Figure 1
+
+4.  Acknowledgements
+
+   To be added if any.
+
+5.  Contributors
+
+   Enrico Griseri
+   Nokia
+
+   Email: Enrico.Griseri@nokia.com
+
+
+   Gabriele Galimberti
+   Cisco
+
+   Email: ggalimbe@cisco.com
+
+
+6.  IANA Considerations
+
+   This memo includes no request to IANA.
+
+   All drafts are required to have an IANA considerations section (see
+   Guidelines for Writing an IANA Considerations Section in RFCs
+   [RFC5226] for a guide).  If the draft does not require IANA to do
+   anything, the section contains an explicit statement that this is the
+   case (as above).  If there are no requirements for IANA, the section
+   will be removed during conversion into an RFC by the RFC Editor.
+
+7.  Security Considerations
+
+   All drafts are required to have a security considerations section.
+   See RFC 3552 [RFC3552] for a guide.
+
+8.  References
+
+8.1.  Normative References
+
+   [I-D.ietf-ccamp-dwdm-if-param-yang]
+              Galimberti, G., Kunze, R., Burk, A., Hiremagalur, D., and
+              G. Grammel, "A YANG model to manage the optical interface
+              parameters for an external transponder in a WDM network",
+              Work in Progress, Internet-Draft, draft-ietf-ccamp-dwdm-
+
+
+
+Beller, et al.           Expires 26 August 2021                [Page 22]
+
+Internet-Draft         Layer 0 Types - Revision 2          February 2021
+
+
+              if-param-yang-05, 18 November 2020,
+              <https://tools.ietf.org/html/draft-ietf-ccamp-dwdm-if-
+              param-yang-05>.
```

Comment:
Since the dwdm-if model will use this mode, I think this reference should be informative